doc/manipulating_visualizing/manipulating_images.rst

```diff
@@ -32,21 +32,25 @@ datasets and atlases. Dataset fetching functions can be imported from
 :mod:`nilearn.datasets`::
 
     >>> from nilearn import datasets
-    >>> haxby_files = datasets.fetch_haxby(n_subjects=1)  # doctest: +SKIP
+    >>> haxby_dataset = datasets.fetch_haxby(n_subjects=1)
```

Comment:
Returning meaningful paths is already what we do in the tests. We can easily generate data in them. However, we sometimes have to generate file with meaningful content (XML) and this is the most difficult part to handle. I mean that we can't automatically mock everything. Some dataset loader requires specific code and we should think of a good way to put that out of the tests  in order to use it elsewhere.